commands/issue/board/view/issue_board_view.go

```diff
@@ -50,64 +50,171 @@ func NewCmdView(f *cmdutils.Factory) *cobra.Command {
+			if boardInfo[selectedBoard].group != nil {
+				boardLists, err = api.GetGroupIssueBoardLists(apiClient, boardInfo[selectedBoard].group.ID,
+					boardInfo[selectedBoard].id, &gitlab.ListGroupIssueBoardListsOptions{})
+				if err != nil {
+					return err
+				}
+			} else {
+				boardLists, err = api.GetPojectIssueBoardLists(apiClient, repo.FullName(),
+					boardInfo[selectedBoard].id, &gitlab.GetIssueBoardListsOptions{})
+				if err != nil {
+					return err
+				}
 			}
 
+			// manually add 'opened' and 'closed' lists before and after fetched lists
+			opened := &gitlab.BoardList{
+				Label: &gitlab.Label{
+					Name:      "Open",
+					Color:     "#fabd2f",
+					TextColor: "#000000",
+				},
+				Position: 0,
+			}
+			boardLists = append([]*gitlab.BoardList{opened}, boardLists...)
+
+			closed := &gitlab.BoardList{
+				Label: &gitlab.Label{
+					Name:      "Closed",
+					Color:     "#8ec07c",
+					TextColor: "#000000",
+				},
+				Position: len(boardLists),
+			}
+			boardLists = append(boardLists, closed)
+
 			root := tview.NewFlex()
 			var issues []*gitlab.Issue
-			// TODO: add `open` and `closed` board list. Both are not returned in the List API response payload
-			for _, list := range boadLists {
-				var boardIssues string
-				issues, err = api.ListIssues(apiClient, repo.FullName(), &gitlab.ListProjectIssuesOptions{
-					State:  gitlab.String("opened"),
-					Labels: gitlab.Labels{list.Label.Name},
-				})
+			for _, list := range boardLists {
+				if list.Label == nil {
+					continue
+				}
+
+				var boardIssues, listTitle, listColor string
+
+				if list.Label == nil {
+					listTitle = "Unnamed"
+					listColor = "#FFA500" // orange
+				} else {
+					listTitle = list.Label.Name
+					listColor = list.Label.Color
+				}
+
+				// automatically request using state for default "open" and "closed" lists
+				// this is required because these lists aren't returned with the board lists api call
+				if list.Label.Name == "Closed" {
+					state = "closed"
+				}
+
+				if list.Label.Name == "Open" {
+					state = "opened"
+				}
+
+				// "closed" and "open" are considered special lists since they
+				// need to be requested using state and not label
+				isSpecialList := list.Label.Name == "Open" || list.Label.Name == "Closed"
+
+				// append list name label to labels from cli
+				reqLabels := labels
+				if reqLabels == "" && !isSpecialList {
+					reqLabels = list.Label.Name
+				}
+				if reqLabels != "" && !isSpecialList {
+					reqLabels = reqLabels + "," + list.Label.Name
+				}
+
+				if boardInfo[selectedBoard].group != nil {
+					opts, err := parseListGroupIssueOptions(assigneeID, assigneeUsername, reqLabels, milestone, state)
+					if err != nil {
+						return err
+					}
+					issues, err = api.ListGroupIssues(apiClient, boardInfo[selectedBoard].group.ID, opts)
+					if err != nil {
+						return err
+					}
+				}
+
+				if boardInfo[selectedBoard].group == nil {
```

Comment:
I did away with all the 'else' statements in the file, I don't know why my mind slipped to add it there as I make an effort to avoid them for clarity.